task/control-plane-agent/src/main.rs

```diff
@@ -339,6 +380,9 @@ impl idl::InOrderControlPlaneAgentImpl for ServerImpl {
         } else {
             UartClient::Mgs
         };
+        ringbuf_entry!(Log::IpcRequest(IpcRequest::SetHumilityUartClient(
```

Comment:
Nit: `SetHumilityUartClient` seems misnamed, since it has the client as a member; how about `SetUartClient`?

Reply:
Gotcha, I wonder if the method predated the `enum UartClient`.  I don't feel too strongly about it!